code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -23,8 +24,10 @@
 /obj/machinery/gibber/RefreshParts()
 	gibtime = 40
 	meat_produced = 0
+	meat_quality = 35 // unupgraded this means quality is 50, and max upgraded it is 95
```

Comment:
might be a good idea to make sure this is clamped between 0 and 100

Reply:
as long as the final product (gibber meat) has its quality set using adjust food quality then it should already be clamped between 0 and 100